TESTING.md

```diff
@@ -309,24 +309,26 @@ func (vcd *TestVCD) Test_ComposeVApp(check *checks.C) {
 }
 ```
 
-# Environment variables
+# Environment variables and corresponding flags
 
 While running tests, the following environment variables can be used:
```

Comment:
We should state which is higher priority too - env vars or flags.

Reply:
Environment variables have precedence.
With this request, you have actually found a bug. Going to fix it